src/main/java/software/sigma/bu003/internship/coparts/client/technomir/exception/ObjectIsEmptyException.java

```diff
@@ -0,0 +1,7 @@
+package software.sigma.bu003.internship.coparts.client.technomir.exception;
+
+public class ObjectIsEmptyException extends RuntimeException {
```

Comment:
EmptypartException also you need to know exact part number and handle it and log it

Reply:
corrected